### PR TITLE
update SyferText.py 

### PR DESCRIPTION
**Pull request by ramesht007**
## Description
Updated versions for SyferText_0.1.0 for issue #139

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -1,31 +1,31 @@
...
+setup(
+    name="syfertext",
+    author="Alan Aboudib",
+    author_email="agabudeeb@gmail.com",
+    description="A privacy preserving NLP framework",
+    url="https://github.com/OpenMined/SyferText",
+    keywords="nlp smpc secure multi-party computation federated learning deep learning artificial intelligence secure model sharin natural language processing spacy spaCy",
+    classifier=["Programming Language :: Python :: 3.6", "Operating System :: OS Independent"],
+    license="Apache-2.0",
+    version="0.0.1",
```
Comment (AlanAboudib) [inline setup.py:14]: What is required is to set the version to 0.1.0.

Would you please apply that?

File: setup.py [hunk 2]
```diff
@@ -1,31 +1,31 @@
...
+    packages=find_packages(),
+    install_requires=[
+        "tqdm==4.47.0",
```
Comment (AlanAboudib) [inline setup.py:17]: Please remove tqdm since we no longer use it

File: setup.py (hunk 1 above)
Comment (ramesht007) [reply setup.py:14]: Ya, sure.........I totally forgot about this😅

File: setup.py (hunk 2 above)
Comment (ramesht007) [reply setup.py:17]: ok

**Comment by ramesht007** [comment]
Thanks @AlanAboudib for giving me the opportunity

**Pull request merged**